dev/scripts/make-update.sh

```diff
@@ -543,16 +543,27 @@ make_stable_update() {
 }
 
 staging_log() {
-  git fetch -q $REMOTE_NAME $STAGING_BRANCH_NAME
-  git log --graph --oneline -U0 --submodule $REMOTE_NAME/$STABLE_BRANCH_NAME..$REMOTE_NAME/$STAGING_BRANCH_NAME | \
-    awk -v version="$STAGING_VERSION" '
-      /^\*.*Staging update [0-9]{8}/ { printf("\n# %s-staging-%s-%s\n", version, $NF, substr($2, 0, 7)) }
-      /^\*/ { printf("  %s\n",$0) }
-      /^  Submodule/ {printf("    %s %s\n", $2, $3)}
-      /^\| Submodule/ {printf("    %s %s\n", $3, $4)}
-      /^    >/ { $1=""; printf("      %s\n", $0 )}
-      /^\|   >/ { $1=""; $2=""; printf("      %s\n", $0 )}
-   '
+  if ! git ls-remote --exit-code --heads $REMOTE_NAME $STAGING_BRANCH_NAME; then
+    echo "staging Branch nicht gefunden, stattdessen wird mit main verglichen"
+    git fetch -q $REMOTE_NAME main
+    git submodule -q foreach git fetch -q $REMOTE_NAME $STABLE_BRANCH_NAME
+    git submodule -q foreach git fetch -q $REMOTE_NAME main
+    echo OpenSlides
+    git log --graph --oneline $REMOTE_NAME/$STABLE_BRANCH_NAME..$REMOTE_NAME/main
+    git submodule -q foreach 'echo $name; git --no-pager log --graph --oneline $REMOTE_NAME/$STABLE_BRANCH_NAME..$REMOTE_NAME/main'
+  else
+    git fetch -q $REMOTE_NAME $STAGING_BRANCH_NAME
+    echo $STAGING_BRANCH_NAME
```

Comment:
Both of these `echo`s are not necessary IMO.
If you want to keep them please make them output the same kind of information, i.e. the branch name being `git log`'d.